### PR TITLE
Adding pd-ssd storage class

### PR DESCRIPTION
**Pull request by aliceh**
In this commit storage calss pd-ssd will be added to all gcp clusters. [OSD-6468](https://issues.redhat.com/browse/OSD-6468)

### CODE REVIEW

File: deploy/osd-storage/config.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+deploymentMode: SelectorSyncSet
+selectorSyncSet: 
+  applyBehavior: "CreateOnly"
```
Comment (dustman9000) [inline deploy/osd-storage/config.yaml:3]: This is so a dedicated-admin user can setup their own private encryption keys in the future.

Comment (aliceh) [reply deploy/osd-storage/config.yaml:3]: Thanks for explaining!

**Review by jharrington22 — APPROVED** [review]
/lgtm

**Pull request merged**